build.gradle

```diff
@@ -185,6 +187,9 @@ ext.libraries = [
 ]
 
 dependencies {
+  implementation group: 'org.postgresql', name: 'postgresql', version: '42.2.10'
+  implementation group: 'org.flywaydb', name: 'flyway-core', version: '6.1.4'
```

Comment:
and this one is `6.3.1`. including the gradle plugin ;)

Reply:
done